controllers/storagecluster/storageclient.go

```diff
@@ -18,8 +18,9 @@ import (
 )
 
 const (
-	ocsClientConfigMapName = "ocs-client-operator-config"
-	manageNoobaaSubKey     = "manageNoobaaSubscription"
+	ocsClientConfigMapName         = "ocs-client-operator-config"
+	manageNoobaaSubKey             = "manageNoobaaSubscription"
+	csiControllerPluginHostNetwork = "csi-ctrlplugin-host-network"
```

Comment:
1. The new key should follow camel casing to align with all existing keys in the config map  
2. Suggesting a new name and value:
```suggestion
	useHostNetworkForCsiControllers = "useHostNetworkForCsiControllers"
```